darwin/dataset/remote_dataset.py

```diff
@@ -42,19 +42,17 @@ def __init__(
             spaces are replaced by dashes, e.g., `bird-species`. This string is unique within a team
         dataset_id : int
             Unique internal reference from the Darwin backend
-        project_id : int
-            [Deprecated] will be removed in next iteration
         image_count : int
             Dataset size (number of images)
         progress : float
             How much of the dataset has been annotated 0.0 to 1.0 (1.0 == 100%)
         client : Client
             Client to use for interaction with the server
         """
+        self.team = team
         self.name = name
         self.slug = slug or name
```

Comment:
this could be dangerous and create failing requests further down the pipeline. Could be fixed with the property comment made above